Jenkinsfile

```diff
@@ -222,6 +243,21 @@ try {
         }
       }
     },
+    'CMake': {
+      node('mxnetlinux-cpu') {
+        ws('workspace/build-cmake-cpu') {
+          init_git()
+          def defines = """ \
+            -DUSE_CUDA=0    \
```

Comment:
Here is a good place for -DCMAKE_BUILD_TYPE=Release if it's not set

Reply:
This build will be packaged and passed on to cpp unit tests (plan to add that next week).  I'd prefer to keep it debug so that we can get non-inlined stacks and assertions failures.

Reply:
@cjolivier01 good point, in my experience a debug build catches more problems than a release build. Of course there are cases where a debug build works and a release one doesn't, those are nasty ones....  Ideally we should run tests in both, but this could be prohibitibely expensive for us.